tests/acceptance/features/apiSharingNg/sharedByMe.feature

```diff
@@ -0,0 +1,424 @@
+Feature: resources shared by user
+  As a user
+  I want to get resources shared by me
+  So that I can know about what resources are shared with others
+
+  https://owncloud.dev/libre-graph-api/#/me.drive/ListSharedByMe
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+  Scenario: user lists the file shared from personal space
+    Given user "Alice" has uploaded file with content "hello world" to "textfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resourceType    | file         |
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Alice" lists the resources shared by her using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should contain file name "textfile.txt" in the item 'value' with the following data:
+    """
+    {
+      "type": "object",
+      "required": [
+        "parentReference",
+        "permissions",
+        "size"
+      ],
+      "properties": {
+        "parentReference": {
+          "type": "object",
+          "required": [
+            "driveId",
+            "driveType",
+            "path"
```

Comment:
<img width="1154" alt="Screenshot 2024-01-29 at 10 50 58" src="https://github.com/owncloud/ocis/assets/84779829/c03fb054-c6f3-43e0-ad29-ef4a43fe4e24">

I asked flo regarding parent name. let wait a little

Reply:
@ScharfViktor sure! but the name is already checked above so i did not double checked it.

Reply:
I assume that is should be parent name

Reply:
@ScharfViktor  the "name" above "parentReference" and inside of it are same.

Reply:
we won't have parentName
please change your tests after merging this PR https://github.com/owncloud/ocis/pull/8307

Reply:
@ScharfViktor sure thanks!